dvc/exceptions.py

```diff
@@ -299,6 +299,16 @@ def __init__(self, code, reason):
         super().__init__(f"'{code} {reason}'")
 
 
+class WebDAVConfigError(DvcException):
+    def __init__(self, host):
+        super().__init__(f"Configuration for WebDAV {host} is invalid.")
+
+
+class WebDAVConnectionError(DvcException):
+    def __init__(self, host):
+        super().__init__(f"Unable to connect to WebDAV {host}.")
+
```

Comment:
These are only used in webdav.py, so let's move them there. Also i'm not sure we need a special WebDAVConfigError exception. We could just raise ConfigError or at least inherit from it.